cob_utilities/package.xml

```diff
@@ -12,12 +12,11 @@
   <maintainer email="mig@ipa.fhg.de">Matthias Gruhler</maintainer>
   <author>Christian Connette</author>
 
-
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roscpp</build_depend>
   <run_depend>roscpp</run_depend>
 
   <export>
-    <cpp lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_utilities" cflags="-I${prefix}/common/include"/>
+    <cpp cflags="-I${prefix}/common/include" lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lcob_utilities"/>
```

Comment:
Do we need those flags at anymore? I guess this is a leftover from `rosbuild`...